src/components/application_manager/test/state_controller/state_controller_test.cc

```diff
@@ -69,12 +69,12 @@ using ::testing::InSequence;
 using ::testing::Truly;
 using ::testing::AtLeast;
 
-static application_manager::MockMessageHelper* message_helper_mock_;
-
 namespace test {
 namespace components {
 namespace state_controller_test {
 
+static application_manager::MockMessageHelper* message_helper_mock_;
```

Comment:
@pvvasilev, than at least move it to test fixture. Other ways it could cause a problems.